Documentation/quickstart.md

```diff
@@ -0,0 +1,49 @@
+# Quickstart - Hubble in 3 Minutes or Less
+
+## 0. Requirements
+
+ * [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+ * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+ * [Helm v3](https://github.com/helm/helm/releases/tag/v3.2.0)
```

Comment:
I think [this link](https://helm.sh/docs/intro/install/) would be better (it points to Helm v3) because v3.2.0 will quickly become obsolete.